app/css/main.css

```diff
@@ -168,7 +178,7 @@ main {
   display: inline-block;
   vertical-align: text-top;
   margin-right: 8px;
-  content: url('data:image/svg+xml; utf8, <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M21.5 12.495L16 18h-3l4.498-4.5H3v-2h14.498L13 7h3z"/></svg>');
+  content: url('data:image/svg+xml; utf8, <svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g id="Artboard" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><polygon id="➔" fill="%23FF4470" points="21.5 12.4953271 16 18 13 18 17.4981689 13.5 3 13.5 3 11.5 17.4981689 11.5 13 7 16 7"></polygon></g></svg>');
```

Comment:
Did the shape change? If not, use the compressed SVG here.

The default arrow has no fill color (i.e. will render as black). Theme-specific SVG with color should go in theme.css.